samza-yarn/src/main/java/org/apache/samza/job/yarn/YarnClusterResourceManager.java

```diff
@@ -202,7 +203,9 @@ public void requestResources(SamzaResourceRequest resourceRequest) {
               capability,
               new String[]{preferredHost},
               null,
-              priority);
+              priority,
+              true,
+              label);
```

Comment:
this breaks compat with older versions. Is there any sort of builder or similar?